tools/strobelight-ctrl/.gitignore

```diff
@@ -0,0 +1 @@
+strobelight-ctrl
```

Comment:
We should pick a different name as this is already in use. Maybe `uprobe-ctl`.

Reply:
will rename it to `probe-ctl`. `uprobe-ctl` is not a good fit, at it could also control kprobes. And as it is nor will be an official OTel artifact, the `otel-` prefix doesn't sound right.

Reply:
renamed with https://github.com/open-telemetry/opentelemetry-ebpf-profiler/pull/662/commits/a0c7545b28f358b619eb6ed2e818b98bcf92f89b